### ISSUE TITLE
Make svg icons perceivable by screen readers

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#866]
Part of #820; Related to #781 and #640 

The svg icons we've added for component type and online content are not perceivable by screen readers. There are a couple of ways to address this.

1. Add `aria-label="[iconname]"` attribute to the `<svg>`

   This seems like the cleaner solution, but I don't know what it implies for the "Used unmodified" credits or if there are i18n implications.

   The attribute could also be attached to a `<span>` surrounding the `<svg>`.

2. Add `<span class="sr-only">[iconname]</span>` after the `<svg>`

Note that neither of these will provide tooltips for sighted users. Nor will adding a `title` attribute, because the icons themselves are not focusable elements. 

### Icon names

| icon | name |
| --- | --- |
| <img width="21" alt="document" src="https://user-images.githubusercontent.com/6945691/65362215-6c120b80-dbbb-11e9-89c2-fc8c9de50da6.png"> | document |
| <img width="24" alt="container" src="https://user-images.githubusercontent.com/6945691/65362212-6c120b80-dbbb-11e9-8f22-97c7fe4cf2fc.png"> | container |
| <img width="26" alt="collection" src="https://user-images.githubusercontent.com/6945691/65362214-6c120b80-dbbb-11e9-9c9a-f5be6e9d2f0e.png"> | collection |
| <img width="30" alt="onlinecontent" src="https://user-images.githubusercontent.com/6945691/65362210-6c120b80-dbbb-11e9-9abb-006ea7a29cde.png"> | online content |
| <img width="30" alt="pdf" src="https://user-images.githubusercontent.com/6945691/65362209-6b797500-dbbb-11e9-9f70-6a67e109f634.png"> | PDF [upper-case important] |
| <img width="30" alt="ead" src="https://user-images.githubusercontent.com/6945691/65362208-6b797500-dbbb-11e9-9d7a-82bfdf4f2d82.png"> | EAD [upper-case important] |

**Comment by mejackreed** [comment]
@jvine what do you think about something like Option 2 upstream in Blacklight so that anytime we use an SVG icon we automatically get this. It could then also be internationalized.

**Comment by jvine** [comment]
It makes sense to do it globally, but won't we also have to do it in Arclight for our local icons? Or does adding it to Blacklight make that easier? Your judgment.

**Comment by mejackreed** [comment]
Yes we will have to do it for our custom icons in Arclight, but it would give us the ability to standardize on a pattern in Blacklight, ArcLight, and GeoBlacklight.

**Comment by mejackreed** [comment]
I've created a ticket upstream with a proposal based of this ticket and past GeoBlacklight experience. https://github.com/projectblacklight/blacklight/issues/2150 I hope we can get some feedback on this approach.

**Comment by mejackreed** [comment]
I think this is resolved with the latest update to Blacklight.
